packages/cli/src/utils/modules/constants.ts

```diff
@@ -0,0 +1,23 @@
+import KeysWithValueModuleData from "@latticexyz/world/out/KeysWithValueModule.sol/KeysWithValueModule.json" assert { type: "json" };
+import KeysInTableModuleData from "@latticexyz/world/out/KeysInTableModule.sol/KeysInTableModule.json" assert { type: "json" };
+import UniqueEntityModuleData from "@latticexyz/world/out/UniqueEntityModule.sol/UniqueEntityModule.json" assert { type: "json" };
+import { ContractCode } from "../utils/types";
+
+// These modules are always deployed
+export const defaultModules: ContractCode[] = [
```

Comment:
for consistency with the other variable names
```suggestion
export const defaultModuleContracts: ContractCode[] = [
```